main.py

```diff
@@ -286,6 +319,44 @@ def validateSettings(staticLetters) -> bool:
     return True
 
 
+def normalized(sett: dict) -> list:
+    total = sum(sett[i] for i in sett)
+    return [(sett[i]/total, i) for i in sett]
+
+
+def orderLetters(configs: tuple = BIGRAMS_CONFIGS) -> str:
+
+    # Generate monograph sets for each language
+    monoSets = []
```

Comment:
Please revisit variable-naming for this function. I find some of names difficult to understand as someone who hasn't written the code. Examples:

- Why is it called `monoSets`? It stores a list of lists of tuples containing ....... characters and their frequencies
- To preserve consistency with functions like `def getBigrams(…)`, please rename `bi` -> `bigram` and `bis` -> `bigrams`
- With the same reasoning, please rename `biFile` to `corpus`
- `notNorm` might more appropriately be called something like "`monograms`"
- `monoSets` might be called "`normalizedMonogramDicts`"
- `combinedSet` might be called "`combinedMonograms`"

Reply:
Done in 444e09cfcf29a2dc6c1ad61332d4f2e98f975ae6